pkg/operation/operation.go

```diff
@@ -134,7 +134,10 @@ func newOperation(
 
 func shootWantsAlertmanager(shoot *gardencorev1alpha1.Shoot, secrets map[string]*corev1.Secret) bool {
 	if alertingSMTPSecret := common.GetSecretKeysWithPrefix(v1alpha1constants.GardenRoleAlertingSMTP, secrets); len(alertingSMTPSecret) > 0 {
-		if address, ok := shoot.Annotations[v1alpha1constants.AnnotationShootOperatedBy]; ok && utils.TestEmail(address) {
+		if shoot.Spec.Monitoring != nil &&
+			shoot.Spec.Monitoring.Alerting != nil &&
+			shoot.Spec.Monitoring.Alerting.EmailReceivers != nil &&
```

Comment:
Think this check `shoot.Spec.Monitoring.Alerting.EmailReceivers != nil` can be removed.

After that it looks good to me. Thanks!

Reply:
Thanks for the review @dkistner! 
I removed `shoot.Spec.Monitoring.Alerting.EmailReceivers != nil`